integrations/gmail/definitions/channels.ts

```diff
@@ -4,7 +4,29 @@ export const channels = {
   channel: {
     title: 'Email thread',
     description: 'Messages in an email thread',
-    messages: sdk.messages.defaults,
+    messages: {
+      ...sdk.messages.defaults,
+      image: {
+        schema: sdk.messages.defaults.image.schema.extend({
+          title: sdk.z.string().title('Alt text').describe('Alt text for the image'),
```

Comment:
Note to reviewers: this override is required because the sdk doesn't define the `title` field for those message types, but it is a mandatory field in the studio